indexer/ai/skill-config.json

```diff
@@ -0,0 +1,67 @@
+{
+  "aiSkillSet" : [
+    {
+      "aiSkillName" : "CloudSearch.StandardAISkills.EntityExtraction",
+      "inputs" : {
+        "language" : "en",
+        "encoding" : "UTF16"
+      },
+      "filters" : {
```

Comment:
lets merge inputs and filters,
you can call below filters as:
minimumSalience:
categories:

Reply:
Okay I will change the filter names, but why merge inputs and filters?